### PR TITLE
Feature connect

### PR DESCRIPTION
**Pull request by dgaleano201611**
Se crea conexión entre el servidor y la base de datos en postgresql llamada restaurantdb, usando prisma, se crea un modelo de prueba llamado user con los campos id, email , name para verificar la creación de la db.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/restaurantdb?schema=public"
+PORT=8080
```
Comment (ht1204) [inline .env:8]: Please, this .env file must not be uploaded and the configurations shall be done in local env according to settings of each developer, add this file in a `.gitignore`

**Review by ht1204 — CHANGES_REQUESTED** [review]
I added a commented related to configurations, please take this to consideration. Thanks in advanced.

File: .env (hunk 1 above)
Comment (dgaleano201611) [reply .env:8]: the .env file inside a gitignore.

**Review by dgaleano201611 — COMMENTED** [review]
the .env file inside a gitignore.

**Review by ht1204 — APPROVED** [review]
LGTM, Good one!

**Review by Nicolas-Rodriguez-Ch — APPROVED** [review]
Looks good to me

**Pull request merged**